test/CMakeLists.txt

```diff
@@ -140,7 +149,7 @@ if(ENABLE_DATA_TOOLS)
   add_dependencies(run-thor_worker utrecht_tiles)
   add_dependencies(run-recover_shortcut utrecht_tiles)
   add_dependencies(run-minbb utrecht_tiles)
-  add_dependencies(run-bidirectional_astar whitelion_tiles roma_tiles)
+  add_dependencies(run-astar whitelion_tiles roma_tiles reversed_whitelion_tiles)
```

Comment:
why not keep both tests for the sake of coverage for both cases?

Reply:
The last commit merges the tests in `bidirectional_astar.cc` with `astar.cc` since the former had evolved to cover more than just bidirectional: https://github.com/valhalla/valhalla/pull/2059/commits/5b3e30bee0a21c948fcf01cbea9e509475c97879

 I didn't see the `astar.cc` when I started the work.